### PR TITLE
Add tutorial notes

### PR DESCRIPTION
**Pull request by Anbranin**
Make em do tutorials

### CODE REVIEW

File: config/qualifications.yml [hunk 1]
```diff
@@ -32,3 +33,85 @@ description: >
...
+  - In section 1.5.1, there are instructions to set up a Heroku account to deploy your code to.
+    Since you'll be deploying code to production in a few short months, you can skip all related steps.
+---
+title: Complete a ruby tutorial
+description: >
+  There are many Ruby tutorials available which you can choose from.
+  Pick one from this list, or if you find one you like better, get approval from your supervisor.
+  - [Learn enough Ruby to be dangerous](https://www.learnenough.com/ruby-tutorial/hello_world)
+  - [Ruby Koans](http://rubykoans.com/)
+  rubymonk(maybe), learn enough Ruby to be dangerous, ruby koans
```
Comment (werebus) [inline config/qualifications.yml]: This looks like perhaps a note to yourself?

File: config/qualifications.yml [hunk 2]
```diff
@@ -32,3 +33,85 @@ description: >
...
+---
+title: Complete a git tutorial (optional)
```
Comment (werebus) [inline config/qualifications.yml]: I think I'd put these before the Rails tutorials. Order doesn't matter _too_ much_ but the issues will be created in this order.

File: config/qualifications.yml [hunk 3]
```diff
@@ -32,3 +33,85 @@ description: >
...
+  Otherwise, consider a tutorial [like this](https://www.learnenough.com/git-tutorial/getting_started)
+---
+title: Complete a command line tutorial (optional)
+description: >
+  Having a good grasp of the command line is essential for working in our team,
+  as we use the command line every single day for various tasks
+  Use [this tutorial](https://www.learnenough.com/command-line-tutorial/basics)
+  or another approved tutorial to brush up on your basics.
+---
+title: Demonstrate understanding of Ruby core methods / Rails object extensions
```
Comment (werebus) [inline config/qualifications.yml]: I feel like I've already reviewed this? #23 perhaps?

File: config/qualifications.yml (hunk 1 above)
Comment (Anbranin) [reply config/qualifications.yml]: sure is buddy

File: config/qualifications.yml (hunk 3 above)
Comment (Anbranin) [reply config/qualifications.yml]: I think this happens when I don't manually update the diff.

Comment (Anbranin) [reply config/qualifications.yml]: Still shows up as new

Comment (werebus) [reply config/qualifications.yml]: Yeah, I missed that #23 wasn't a PR into master.

File: config/qualifications.yml (hunk 2 above)
Comment (werebus) [reply config/qualifications.yml]: Also the command line?

**Comment by Anbranin** [comment]
Why is my build broken

**Comment by werebus** [comment]
Man, am I tired of Bundler bullshit on Travis.

I added (a0f4b937c4aa3f92eb87b71637c7769065af27ee) steps to update Bundler in CI, but since then Bundler 2.0.2 was released, I guess?

And it's like, "hey, hey, hey, you said you need bundler **2.0.1**. What R U doing?"

**Comment by Anbranin** [comment]
Well so I can change the `.travis.yml` file to have bundler 2.0.1 OR I can bundle update --bundler? I think that will also work. Will take longer. Also we'd have to keep the BUNDLED WITH updated as well.

**Comment by Anbranin** [comment]
@werebus I think I've addressed all your issues, if you'd like to take another look. Also, wondering if I did the right thing for Travis.

**Comment by werebus** [comment]
First, I think #24 (just opened) is a bit better fix for the Bundler version issue.

Second, I'm not 100% sure I agree with, "all of the issues are technically optional if one can demonstrate competence." For "go off and do a thing" type issues, you aren't really asking for that demonstration.

**Comment by Anbranin** [comment]
OK, what do you suggest about the go-off-and-do-a-thing type issues? I'm now leaning towards making none of them optional. My idea is that to close the issue, include a link to which tutorial or whatnot you did. If you're more competent, you can do some advanced reading. Just something. I could include suggestions.

File: config/qualifications.yml [hunk 4]
```diff
@@ -33,3 +34,87 @@ description: >
...
+  and what logic does _not_ go in a view.
+  - Use a partial
+  - Demonstrate understanding of how to use routes (a link, a redirect, or creating routes)
+  - Use params in a controller
+---
+title: Demonstrate understanding of correct and proper git usage
+description: >
+  Show that you understand how to properly use Git by writing commits that
+  generally relate to atomic features or thoughts. Make sure you know how to
+  resolve a merge conflict correctly.
```
Comment (LiamBrandt) [inline config/qualifications.yml]: How are you supposed to show that you know how to resolve a merge conflict correctly? Just ask somebody if what you are doing is correct?

File: config/qualifications.yml [hunk 5]
```diff
@@ -33,3 +34,87 @@ description: >
...
+  Tag some related issues.
+---
+title: have given feedback on another programmers code
+description: >
+  Ask helpful questions/suggest alternatives on someone else's code
+---
+title: display a good understanding of how and why to write tests
+description: >
+  Demonstrate knowledge of basic model/controller/system test structure.
+  Demonstrate understanding of mocks/stubs and what purpose they serve.
```
Comment (LiamBrandt) [inline config/qualifications.yml]: How does a trainee demonstrate to themselves that they have this knowledge or understanding? Should they write tests for some part of an app?

Comment (Anbranin) [reply config/qualifications.yml]: Yes. "demonstrate" in most cases means "write code that uses"

**Comment by Anbranin** [comment]
@werebus, I'd _prefer_ not to without your express approval. I think at this point I've addressed all the comments.
I'd like to actually use this for the current batch of trainees because I think real life feedback is desirable at this point.

**Pull request merged**